app/saqib/page.js

```diff
@@ -27,42 +44,50 @@ export default function TodoApp() {
         <div>
           <h1 className="header">To-do List ✏️</h1>
         </div>
-        <div className="form-body">
-          <form id="todo-form" onSubmit={addTask}>
+        {/* form */}
+        <div className="form-body text-align: center;">
+          <form onSubmit={addTask}>
             <input
               type="text"
-              id="new-todo"
-              // value={newTask}
+              value={newTask}
               onChange={(e) => setNewTask(e.target.value)}
-              placeholder="New Task..."
+              placeholder="Add task"
             />
-
-            <button type="submit" id="submit-new" className="button1">
+            <button type="submit" className="add">
               Add
             </button>
           </form>
         </div>
+
         {/* Delete all task */}
         <div className="delete-all">
-          <button type="button" id="delete-completed" className="button2">
-            Delete all
-          </button>
+          <button onClick={() => deleteAllItems(todos)}>Delete all</button>
         </div>
+
+        {/* display todo list */}
         <div>
-          <ul className="todo-list">
-            {/* display todo task */}
-            {/* {todos && todos.length ? "" : "No task.."} */}
-            {todos.map((todo) => (
+          {todos.map((todo, index) => {
+            return (
               <li
                 className={todo.completed ? "line-through" : ""}
                 key={todo.id}
+                onClick={(e) => clickHandler(todo)}
               >
                 {todo.task}
+                <button onClick={() => deleteItem(todo.id)} className="delete">
```

Comment:
this does not work because the `onClick` on the parent `<li>` is taking priority over the button's `onClick` if you move the parent's `onClick` into a `<span>` like this then it should work:
```js
                <span onClick={(e) => clickHandler(todo)}>
                  {todo.task}
                </span>
```